packages/codegen/generate.server.utils.js

```diff
@@ -335,6 +335,11 @@ function _generateServerUtilsDeprecated (gql) {
     }
 }
 
+function wrapIfNeeded (fields) {
```

Comment:
Please don't add `IfNeeded` to function name, let it be `IfMissing`

Maybe `wrapInBracesIfMissing`/`addBracesIfMissing` or `...CurlyBraces...` would be better?